Multitouch Support/Native/VoodooI2CNativeEngine.cpp

```diff
@@ -137,14 +139,32 @@ bool VoodooI2CNativeEngine::isForceClickEnabled() {
         lastForceClickPropertyUpdateTime = now_abs;
         return lastIsForceClickEnabled;
     }
-
+    
     // Blocking reading here takes about 10 microseconds
     OSDictionary* dict = OSDynamicCast(OSDictionary, this->getProperty("MultitouchPreferences", gIOServicePlane, kIORegistryIterateRecursively));
     if (!dict) {
         return lastIsForceClickEnabled;
     }
     if (OSCollectionIterator* i = OSCollectionIterator::withCollection(dict)) {
         while (OSSymbol* key = OSDynamicCast(OSSymbol, i->getNextObject())) {
+            // If tap to click (property "Clicking") is disabled,
+            // then we should forbid force clicking,
+            // because it makes impossible to perform a non-force click
+            if (key->isEqualTo("Clicking")) {
```

Comment:
Nitpick: There is no guarantee that the property will not change in the future. We can refactor this method to handle any such situations in the future.

Stylistic: Can we perhaps pull the string `"Clicking"` into a `#define` constant?

Reply:
As other code in this method isn't using consts for keys IDK if you really need to do it now...